indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
Yes, I already have the change for that on my git stash (and iirc also mentioned in the PR where I added the sass-lint config). But maybe let's wait either until we rebase devel on top of master the next time or until we merge devel into master to avoid having to do it in two places?